.phan/config.php

```diff
@@ -30,7 +30,7 @@
 		"PhanUndeclaredClassMethod",
 		"PhanUndeclaredTypeParameter",
 		"PhanUndeclaredConstant",
-		"PhanTypeMismatchForeach",
+		"PhanTypeMismatchDefault",
```

Comment:
This looks like it's just trading one ignored type for another. Why is MismatchDefault re-added?

Reply:
Oops bad rebase. I'll re-delete it.